systemtest/src/main/java/io/strimzi/systemtest/Resources.java

```diff
@@ -574,9 +578,9 @@ private KafkaTopicBuilder defaultTopic(String clusterName, String topicName, int
                 .endSpec();
     }
 
-    DoneableKafkaTopic topic(KafkaTopic topic) {
+    DoneableKafkaTopic topic(KafkaTopic topic, String namespace) {
```

Comment:
Why was this change necessary? Can't we assume that the given `topic` has the namespace set already, and throw if not?